src/middleware/packages/crypto/verifiable-credentials/vc-issuer-service.js

```diff
@@ -0,0 +1,196 @@
+const { MIME_TYPES } = require('@semapps/mime-types');
+const path = require('node:path');
+const jsigs = require('jsonld-signatures');
+const VCCredentialContainer = require('./vc-credential-container');
+
+const {
+  purposes: { AssertionProofPurpose }
+} = jsigs;
+
+let cryptosuite;
+let DataIntegrityProof;
+let vc;
+/** @type {import('@digitalbazaar/ed25519-multikey')} */
+let Ed25519Multikey;
+(async () => {
+  vc = await import('@digitalbazaar/vc');
+
+  ({ cryptosuite } = await import('@digitalbazaar/eddsa-rdfc-2022-cryptosuite'));
+  ({ DataIntegrityProof } = await import('@digitalbazaar/data-integrity'));
+  Ed25519Multikey = await import('@digitalbazaar/ed25519-multikey');
+})();
+
+const { KEY_TYPES, credentialsContext } = require('../constants');
+
+/**
+ * Service for verifying, reading, and revoking Verifiable Credentials.
+ *
+ * WARNING: Changing things here can have security implications.
+ *
+ * @type {import('moleculer').ServiceSchema}
+ */
+const VCCredentialService = {
+  name: 'crypto.vc.issuer',
+  settings: {
+    vcApiPath: null,
+    podProvider: null
+  },
+  created() {
+    const { vcApiPath, podProvider } = this.settings;
+    this.broker.createService({
+      mixins: [VCCredentialContainer],
+      settings: { path: path.join(vcApiPath, 'credentials'), podProvider }
+    });
```

Comment:
I don't think it's a good idea to use such a long path (`http://localhost:3000/test77/data/api/vc/v0.3/credentials`). In the future, we'll probably only have one-level container hierarchy. I suggest to use simply `/credentials`.

Reply:
Also, I would suggest to create this service also on the main VC service, to make the code easier to read.

Reply:
OK. But soon all containers will have UUID so these long paths will disappear anyway.

Reply:
For those kinds of cases, it might be good to set up "redirects" in the controlled container mixin?